notesapplication/notes/api_v1/pagination.py

```diff
@@ -0,0 +1,17 @@
+from notes.api_v1.serializers import NoteVersionSerializer
+from rest_framework import pagination
+from rest_framework.pagination import PageNumberPagination
+
+
+class CustomPagination(pagination.PageNumberPagination):
+    """
+    Class for pagination of comments and versions Viewsets
+    """
+
+    def pagination(queryset, request, pk=None):
+        print("in pagination")
+        paginator = PageNumberPagination()
+        paginator.page_size = 10
```

Comment:
Either add this in `settings.py` as the default value or describe it as a constant. I think the first approach is more apt.